macosx/HBAVPlayer.m

```diff
@@ -12,12 +12,12 @@
 static void *HBAVPlayerRateContext = &HBAVPlayerRateContext;
 static void *HBAVPlayerItemStatusContext = &HBAVPlayerItemStatusContext;
 
-typedef void (^HBRateObverser)(void);
-typedef void (^HBPlayableObverser)(void);
+typedef void (^HBRateObserver)(void);
+typedef void (^HBPlayableObserver)(void);
```

Comment:
I'm hoping that changing this file isn't a problem, I'm less comfortable w/ Cocoa and similar languages (and am not actively looking at the code to consider whether something is public/private/possibly used outside the repo)